src/goBrowsePackage.ts

```diff
@@ -26,64 +26,73 @@ export function browsePackages() {
 		selectedText = getImportPath(selectedText);
 	}
 
-	if (isGoListComplete()) {
-		return showPackages(selectedText);
+	showPackageFiles(selectedText);
+}
+
+function showPackageFiles(pkg: string)  {
```

Comment:
if `pkg` is empty string which can happen when the command is run without any editor open, then we will be needlessly creating a process to run `go list`